source/ambient.d.ts

```diff
@@ -0,0 +1,8 @@
+declare module "node-fetch";
+declare module "parse-diff";
+declare module "lodash.includes";
+declare module "lodash.find";
+declare module "jest-runtime";
+declare module "jest-environment-node";
+
+declare module "*/package.json";
```

Comment:
Ooh, does this mean one doesn’t have to change the syntax used like so? https://github.com/alloy/relational-theory/pull/16/files#diff-e4650f225ab079757ad601f52c84284fR8